docs/deep-linking.md

```diff
@@ -40,12 +40,50 @@ const SimpleApp = createAppContainer(createStackNavigator({...}));
 // on Android, the URI prefix typically contains a host in addition to scheme
 // on Android, note the required / (slash) at the end of the host property
 const prefix = Platform.OS == 'android' ? 'mychat://mychat/' : 'mychat://';
+// in Expo projects you can do the following to get the URI prefix
+const prefix = Expo.Linking.makeUrl('/');
```

Comment:
we might want to have an entirely separate section for expo projects / vanilla react native projects, and just repeat this entire code example in both sections but with the prefix line that is relevant to the platform. it's a bit cleaner when you can just pick the section relevant to you and follow that

Reply:
yes, thanks! I'll do that.